test/onnx/dynamo/test_exporter_api.py

```diff
@@ -190,6 +190,7 @@ def test_raise_on_invalid_save_argument_type(self):
             onnx.ModelProto(),
             io_adapter.InputAdapter(),
             io_adapter.OutputAdapter(),
+            io_adapter.OutputAdapter(),
```

Comment:
this looks weird

Reply:
oh i see you are adding another output adapter in this PR.

Reply:
yes, we need one output adapter to go from torch to onnx format and another to go from onnx to torch.
e.g. onnx->torch we drop extra outputs on the onnx model; torch->onnx we prepend outputs into the torch output